reason-react-native/src/apis/PanResponder.md

```diff
@@ -1,9 +1,137 @@
 ---
-id: apis/Packager
-title: Packager
-wip: true
+id: apis/PanResponder
+title: PanResponder
 ---
 
+`PanResponder` allows more fine-grained control of the _Gesture Responder
+System_, as the handlers can depend also on `gestureState`, in addition to the
+`Event.pressEvent` which are passed to the _Gesture Responder System_ handlers.
+In essence, by means of a custom `PanResponder`, _Gesture Responder System_
+handlers are extended to also depend upon the `gestureState`.
+
+## Methods
```

Comment:
Maybe before directly listing method, we should explain who to pass those? I think the block below with the example could be better before this listing.

Reply:
Should the order be
• Types
• Methods
• Example
then?

An example could be hard to make sense of without an idea of the types and methods

Reply:
Yeah depending on the thing an example can indeed be hard to digest.
This order is ok for me. A "textual transition" can even explain it if necessary :)